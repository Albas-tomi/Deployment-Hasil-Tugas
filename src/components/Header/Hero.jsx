import React from 'react'

function Hero({children}) {
  return (
    <div className='bg-gray-100 flex flex-col items-center h-28 text-center'>
        <p className=' text-2xl font-bold mx-auto flex flex-col p-4'>{children}</p>
    </div>
  )
}

export default Hero