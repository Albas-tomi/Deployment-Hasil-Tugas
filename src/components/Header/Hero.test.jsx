import Hero from "./Hero"; 
import renderer from 'react-test-renderer'

describe('Hero.jsx', () =>{
    it('should correcly', () =>{
        const tree = renderer.create(<Hero/>).toJSON();
        expect(tree).toMatchSnapshot();
    })
})