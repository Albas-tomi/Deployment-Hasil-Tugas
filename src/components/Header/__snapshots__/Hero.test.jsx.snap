// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Hero.jsx > should correcly 1`] = `
<div
  className="bg-gray-100 flex flex-col items-center h-28 text-center"
>
  <p
    className=" text-2xl font-bold mx-auto flex flex-col p-4"
  />
</div>
`;
